templates/base/server/utils/core/logging/logger.py

```
@@ -0,0 +1,106 @@
+import atexit
+import datetime as dt
+import json
+import logging.config
+import logging.handlers
+from pathlib import Path
+from typing import override
```

Pylint (3.3.7)

[error] No name 'override' in module 'typing'

(E0611)